### ISSUE TITLE
let-alist out of date

### ISSUE THREAD

**Issue by zlalanne** [d12frosted/elpa-mirror#9]
Looks like this mirror only has gnu/let-alist 1.0.5 whereas ELPA has 1.0.6 (added back in July)

https://elpa.gnu.org/packages/let-alist.html

**Comment by d12frosted** [comment]
Hey, thank you for the report.

For some reason ELPA packages are outdated. Not only the one you linked. Will fix that.

**Comment by d12frosted** [comment]
I think my `rsync` link is not right.

**Comment by d12frosted** [comment]
Tried different `rsync` links, didn't work. This issue actually resembles #3. That time I just manually upgraded GNU ELPA packages, this time I switched to `https` completely. Though it a) increased build time and b) bombards server more than it should. So I will try to find out a better solution.

But this particular issue is fixed by 62ce0d5a5cb5b62a08f95537e3ad0e55afdac71d (https://github.com/d12frosted/elpa-mirror/commit/62ce0d5a5cb5b62a08f95537e3ad0e55afdac71d#diff-090eb998834c3305ccaec770d0faa932)

Thanks again for reaching me out!
